doc/getting-started.md

```diff
@@ -41,6 +44,11 @@ To build and run Klimatkollen, some dependencies are required apart from Node.js
 
 The full list is in "package.json" which is a file that npm can understand. To have it install all of the dependencies at the right versions, run `npm ci`.
 
+## Our frontend technologies
+
+* [Astro](https://docs.astro.build/en/tutorial/0-introduction/) - our meta framework that handles routing, data and content. Astro has support for all different frontend frameworks which makes it flexible to work with.
+* [Svelte](https://learn.svelte.dev/tutorial/welcome-to-svelte) - components for interactive UI and data visualizations (that will be implemented). In addition to this, we will probably reuse some React-components from our existing frontend.
+
```

Comment:
Okay, thank you! I removed the 'front-end technologies' section from `getting-started.md` since it was intended for Klimatbryan/beta and also fixed a some typos I happened to notice in `contributing.md` at the same time.